cmd/dmsg-discovery/internal/api/api.go

```diff
@@ -142,6 +147,64 @@ func (a *API) getEntry() func(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+// allEntries returns all client entries connected to dmsg
+// URI: /dmsg-discovery/entries
+// Method: GET
+func (a *API) allEntries() func(w http.ResponseWriter, r *http.Request) {
+	return func(w http.ResponseWriter, r *http.Request) {
+		entries, err := a.db.AllEntries(r.Context())
+		if err != nil {
+			a.handleError(w, r, err)
+			return
+		}
+		a.writeJSON(w, r, http.StatusOK, entries)
+	}
+}
+
+// deregisterEntry delete entries of a PK from by network monitor request
```

Comment:
Since we are just deleting one entry it should be `deregisterEntry deletes the entry of a PK requested by the network monitor`

Reply:
Done